From fffac8dbc63882595f2cc35deee7836e444f3d6f Mon Sep 17 00:00:00 2001
From: Sam Mortimer <sam@mortimer.me.uk>
Date: Wed, 8 Aug 2018 23:27:12 -0700
Subject: [PATCH 05/13] lineage-sdk: update lineage_platform_res for pie

intermediates are now put in different subdirs depending
on what generated them.

Change-Id: I1c276bb6156162d02904721c8c0aee9d5c065c83
---
 Android.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Android.mk b/Android.mk
index c546c9c..6dbbc59 100644
--- a/Android.mk
+++ b/Android.mk
@@ -22,7 +22,7 @@ LOCAL_PATH := $(call my-dir)
 # did, the PRIVATE_* vars for R.java wouldn't be guaranteed to be correct.
 # Instead, it depends on the R.stamp file, which lists the corresponding
 # R.java file as a prerequisite.
-lineage_platform_res := APPS/org.lineageos.platform-res_intermediates/src
+lineage_platform_res := APPS/org.lineageos.platform-res_intermediates/aapt
 
 # List of packages used in lineage-api-stubs
 lineage_stub_packages := lineageos.app:lineageos.content:lineageos.hardware:lineageos.media:lineageos.os:lineageos.preference:lineageos.profiles:lineageos.providers:lineageos.platform:lineageos.power:lineageos.util:lineageos.weather:lineageos.weatherservice:lineageos.style:lineageos.trust
-- 
2.17.1

